### PR TITLE
Claire Caudwell Uplifting quotes mobile app

### PR DESCRIPTION
**Pull request by ClaireCaudwell**
**My reflections on how this week's project turned out:**
Bit of a tough week. In the beginning I had a specific idea in mind, but was struggling with if my idea will work/if I can implement it in react native. This was down to the fact that I didn't know react native. That being said I read the documentation, watched a few videos and got some advice from others on how it could work. In the end I managed to achieve my goal and learnt a bunch of react native things along the way.  If you want to read more on what I achieved etc you can read my [README](https://github.com/ClaireCaudwell/project-react-native-app) 😊

**Things I'd like to have clarified or explained in more detail:**
How one can create a reusable component that's a link between pages which you can insert into another component. Also how I can make my code less but still have the outcome I have now.

I managed to upload my github repo to a snack on expo.io [here](https://snack.expo.io/@clairecaudwell/github.com-clairecaudwell-project-react-native-app) 
When viewing my project in the snack some of the proportions are different to how I saw them in chrome with the iphone X view on in the devtools. The best way to view it from the expo snack is:

📱 On a iphone X if you scan the VR code OR 💻 In the web option and be sure to click on the open in new browser icon and make the screen slightly bigger dragging the bottom right corner.❤

### CODE REVIEW

**Review by EmOlsz — CHANGES_REQUESTED** [review]

File: App.js [hunk 1]
```diff
@@ -1,26 +1,66 @@
...
 
-const Container = styled.View`
-  flex: 1;
-  background-color: papayawhip;
-  justify-content: center;
-  align-items: center;
-`
+import QuoteList from './components/QuoteList';
+import Heart from './components/Heart';
+import Sun from './components/Sun';
```
[the remark this answers is not in the document]
Comment (ClaireCaudwell) [reply App.js:8]: Hi @EmOlsz 😊 Yes I know. Initially I did have it as one component where I was mapping over the array of each quote. But the problem I had was that I wanted to place each of my icons, sun, heart, hand etc in a specific place. So I had to create a component for each of them! Before I was getting a grid like layout, but I wanted to customise it. I think I wrote about this in my read me and maybe I can arrange a 1:1 to get a advice if there is any other way of placing the icons specifically without having to have multiple components that are almost identical.

File: components/Bird.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { SafeAreaView, StyleSheet, ScrollView } from 'react-native';
+import styled from 'styled-components/native';
+
+import BackgroundImage from './icons/Background.png';
```
[the remark this answers is not in the document]
Comment (ClaireCaudwell) [reply components/Bird.js:5]: Noted 👍

File: components/Bird.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    font-size: 18px;
+    padding: 10px;
+    color: rgb(255, 255, 255);
+    margin-top: 10px;
+    font-weight: 500;
+`;
+
+const styles = StyleSheet.create({
+    container: {
+      flex: 1,
```
[the remark this answers is not in the document]
Comment (ClaireCaudwell) [reply components/Bird.js:91]: Ok perfect and noted. Thanks for the feedback 👍

File: components/QuoteList.js [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+    return ( 
+        <SafeAreaView style={styles.container}>
+            <ScrollView>
+                <ViewContainer>
+                    <TextContainer>
+                        <TitleText>UPLIFTING QUOTES</TitleText>
+                        <SubtitleText>Click on a symbol to read a quote</SubtitleText>
+                    </TextContainer>
+                    <ButtonContainer>
+                        <HeartButton onPress={()=> navigation.navigate('Heart')}>
```
[the remark this answers is not in the document]
Comment (ClaireCaudwell) [reply components/QuoteList.js:28]: @EmOlsz Yep I agree, but see my comment to your first comment about what I was trying to achieve 😊

**Review by EmOlsz — APPROVED** [review]

**Pull request closed without merging**